### ISSUE TITLE
Road to fully functional Android Client/ios release

### ISSUE THREAD

**Issue by madeofpendletonwool** [madeofpendletonwool/PinePods#320]
At this point android clients are automatically generated on each release, however, there are some issues to work out yet before I'm comfortable fully releasing an official client to a store front like F-droid. This issue will be used to monitor progress to a final client release.

Because this is built with Tauri 2.0 the ios release can come at any time the Android Client is fully ready to go as well.

Issues Known: 

- [x] Fix chrome full episode screen time slider, color wrong and on chrome the tab doesn't line up
- [x] On mobile remove refresh and hamburger menu when full screen. Just send the drop to the top
- [x] On mobile make the action buttons larger
- [x] On mobile make the volume slider drop below shownotes
- [x] When pod is playing play and skip are too far left, also make image raise full screen as well as title. 
- [x] Drag to raise player
- [x] On mobile long press to add to saved etc...?
- [x] On mobile local Downloads page is broken - doesn't open
- [x] Mobile loading screen causes search to jump to right and playing pod to go too low
- [x] On mobile nextcloud doesn't redirect back after adding
- [x] mobile version giving one of two share links wrong. Provides the current server - which on mobile is wrong
- [x] Issues with clicking buttons on android app after episode starts playing. Kinda freezes the whole app. Buttons no longer interactive
- [x] Local Downloading an episode doesn't currently work. More investigation needed. Potentially need android compiling alternative to reqwest to download the episodes.
- [x] Android metadata interaction doesn't work with the APIs in app. So this means Android is unaware of play/pausing and cannot do that interaction. This is specific to the app and not an issue in the web version

**Comment by madeofpendletonwool** [comment]
Fixed the crashing after playing an episode. This was due to an unwrap occurring on a function not available to the mobile app specifically meant to browser API access. I now just safely handle that unwrap

**Comment by madeofpendletonwool** [comment]
Yep, it's coming. That's been a hotly requested feature. Along with carplay.

**Comment by sttrebo** [comment]
Any chance for a manual reorder of the feed or Playlist items?
